.github/workflows/ci.yml

```diff
@@ -475,6 +475,89 @@ jobs:
           path: ${{ github.workspace }}/out/*.zip
           name: llama-box-linux-${{ matrix.arch }}-cann-${{ matrix.version }}
 
+  linux-musa:
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [ amd64 ]
+        version: [ '1.5.1-mp_22' ]
```

Comment:
can we use a '{major}.{minor}' version here? and then map below, as other jobs did?

just like `version: [ '1.5' ]`

Reply:
Done.